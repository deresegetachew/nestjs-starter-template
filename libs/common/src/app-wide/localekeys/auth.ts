export enum authErrorLocaleKey {
    invalidCredential = "common:error.invalid-credentials",
    internalServerError = "common:error.internal-server-error",
    emailTaken = "common:error.email-taken",
    passwordLengthToShort = "common:error.password-to-short",
};



export enum authMessageLocaleKey {
    welcome = "common:message.welcome-login",
    successfullySignedUp = "common:message.successfully-signed-up",
    confirmationEmailSentToYourAccount = "common:message:account-confirmation-email-sent-to-your-account",
}