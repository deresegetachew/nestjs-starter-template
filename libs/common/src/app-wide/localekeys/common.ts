export const enum commonErrorLocaleKey {
    internalServerError = "common:error.internal-server-error",
    notFoundError = "common:error.not-found"
}
