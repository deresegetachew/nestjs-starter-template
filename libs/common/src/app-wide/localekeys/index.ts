export * from '../../class-validator/localekeys';
export * from './common';

