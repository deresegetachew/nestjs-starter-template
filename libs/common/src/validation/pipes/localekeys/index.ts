export enum localeKey {
    parseUUID = 'common:validation.parse-uuid',
    parseint = 'common:validation.parse-int',
    parseBool = 'common:validation.parse-bool',
    parseArray = 'common:validation.parse-array'
}