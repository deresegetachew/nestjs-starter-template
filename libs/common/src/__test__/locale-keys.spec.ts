import { commonErrorLocaleKey } from "../app-wide/localekeys";

describe('commonErrorLocaleKey', () => {
    it('matches enum definition snapshot', () => {
        const keys = JSON.stringify(commonErrorLocaleKey);
        expect(keys).toMatchSnapshot();
    })
})