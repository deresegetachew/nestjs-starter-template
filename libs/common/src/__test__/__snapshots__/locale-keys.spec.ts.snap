// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commonErrorLocaleKey matches enum definition snapshot 1`] = `"{\\"internalServerError\\":\\"common:error.internal-server-error\\",\\"notFoundError\\":\\"common:error.not-found\\"}"`;
